### ISSUE TITLE
Auto Recruiter counter

### ISSUE THREAD

**Issue by GroupTherapyPT** [OpenThrone/OpenThrone#268]
Can I request for a counter?

The recruiter is fiddly at best, sometimes it gives some errors like a session timeout or "error fetching new user" that one might mistake for the one where there are "no valid users for recruitment".

And honestly I hope you're looking for a more expedite solution for the recruiter or at least considering lowering the daily recruit limit, because leaving a recruiter tab open might not be an inconvenience for most of the white collars, but for us menial taskers scheduling 2 hours for it gets rough.

**Comment by uaktags** [comment]
Absolutely, however the reason i've been reluctant to add one is due to the fact that we utilize multiple windows. So you may see 350, 348, 346, etc due to have 2 simultaneous windows running. but i'll add it to my list to things to test on implementing.

Auto-recruiter though is being removed as its currently implemented. Later on it'll only be an Alliance specific feature, something to make clicks within the alliance easier. Recruiting as a whole is more of a game-revenue function than anything else: yes it helps grow your army faster, but the value is more for placing ads on a page to get views and hopefully conversions. 

For solely the purpose of growing your army, the auto-recruiter is kind of a cheat built first party natively as an alternative to needing archaic things like ClickMaster solutions :-)

**Comment by GroupTherapyPT** [comment]
I totally get you but as is, I'd rather click the damn button for 25 minutes, as God intended, than do with all the waiting :')
It's fine knowing it is a stop gap though.

**Comment by uaktags** [comment]
Yea, i think the roll out is going to be first to implement the daily limit of like 200-ish, have that be tied to a Structure Upgrade to get more clicks, and then later remove the auto-recruiter. It'll limit how much clicking is needed as we slowly shift to what the final product will be
